mod/ckeditor/views/default/elgg/ckeditor.js

```diff
@@ -13,7 +13,7 @@ define(function (require) {
 	require('elgg/init');
 	var $ = require('jquery');
 	require('jquery.ckeditor');
-	var CKEDITOR = require('ckeditor');
+	var CKEDITOR = require('ckeditor/ckeditor');
```

Comment:
If you swap this and the next line you will get the same effect as above. Could this be the pitfall?